models/nxos/acls/nxos_acls.yml

```diff
@@ -0,0 +1,395 @@
+---
+GENERATOR_VERSION: '1.0'
+
+ANSIBLE_METADATA: |
+    {
+      'metadata_version': '1.1',
+      'status': ['preview'],
+      'supported_by': 'network'
+    }
+NETWORK_OS: nxos
+RESOURCE: acls
+COPYRIGHT: Copyright 2019 Red Hat
+
+DOCUMENTATION: |
+  module: nxos_acls
+  version_added: '2.10'
+  short_description: Manage named IP Access Control Lists on NX-OS platform
+  description: Manage named IP Access Control Lists on NX-OS platform
+  author: Bradley Thornton (@cidrblock) & Adharsh Srivats Rangarajan (@adharshsrivatsr)
+  notes:
+    - Tested against NX-OS 7.3.(0)D1(1) on VIRL
+    - <documentation for sequence number>
+    - To parse configuration text, provide the output of show running-config | section access-list or a mocked up config
+  options:
+    running_config:
+      description:
+        - Used to parse given commands into structured format, only in parsed state
```

Comment:
```suggestion
        - Parse given commands into structured format. Required if I(state=parsed).
```

Reply:
please add periods to the end of all descriptions here.

Reply:
I(state=parsed) --> what is this?